packages/fcl-wc/src/fcl-wc.js

```diff
@@ -0,0 +1,38 @@
+import SignClient from "@walletconnect/sign-client"
+import {makeServicePlugin} from "./service"
+export {getSdkError} from "@walletconnect/utils"
+
+const DEFAULT_RELAY_URL = "wss://relay.walletconnect.com"
+const DEFAULT_LOGGER = "debug"
+const DEFAULT_APP_METADATA = {
+  name: "FCL WalletConnect",
+  description: "FCL DApp for WalletConnect",
+  url: "https://flow.com/",
+  icons: ["https://avatars.githubusercontent.com/u/62387156?s=280&v=4"],
+}
+
+const initClient = async ({projectId, metadata}) => {
+  if (typeof projectId === "undefined") {
```

Comment:
Can you have a null projectId?

Reply:
yeah, funny thing is you can. Despite them saying it's required